src/utils/config.ts

```diff
@@ -15,3 +15,14 @@ export type SchemaConfig = {
   contentType?: "text" | "image";
   context?: string; //the collection id to get data from
 };
+
+export const modelProviderOptions = {
+  openai: ["gpt-4o-mini", "gpt-4o", "gpt-4", "gpt-3.5-turbo", "Custom"],
+  anthropic: [
+    "claude-3-5-sonnet-20240620",
+    "claude-3-opus-20240229",
+    "claude-3-sonnet-20240307",
+    "claude-3-haiku-20240307",
+  ],
+  groq: ["llama3-8b-8192", "llama3-70b-8192", "llama3-70b-4096"],
+};
```

Comment:
moved modelProviderOptions to config so whenever we add a provider or model we don't need to update multiple files